index.html

```diff
@@ -10,951 +12,1033 @@
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fab fa-superpowers"></i>
                   </span>
                   <span>Examples</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fa fa-book"></i>
                   </span>
                   <span>Documentation</span>
-                </a>
+                </button>
               </span>
               <span class="navbar-item">
-                <a class="button is-white is-outlined">
+                <button class="button is-white is-outlined">
                   <span class="icon">
                     <i class="fab fa-github"></i>
                   </span>
                   <span>View Source</span>
-                </a>
+                </button>
               </span>
-            </div>
+            </nav>
           </div>
         </div>
       </div>
     </div>
 
-    <div class="hero-body">
+    <section class="hero-body">
       <div class="container has-text-centered">
         <div class="column is-6 is-offset-3">
-          <div class="title has-text-white">We have web accessibility in mind</div>
-          <div class="subtitle has-text-white">
+          <h1 class="title has-text-white">We have web accessibility in mind</h1>
+          <h2 id="subscribtion" class="subtitle has-text-white">
             Would you like to get the latest news?
-          </div>
-          <div class="box">
-            <div class="field is-grouped">
-              <p class="control is-expanded">
-                <input class="input">
-              </p>
-              <p class="control">
-                <a class="button is-info">Notify Me</a>
-              </p>
-            </div>
-          </div>
+					</h2>
+          <form aria-labelledby="subscribtion" class="box">
+            <fieldset class="field is-grouped">
+							<legend class="visuallyhidden">You can enter email address for email subscribtion and click "Notify Me" button.</legend>
+              <label for="emailInput" class="control is-expanded visuallyhidden">Enter your email</label>
+              <input aria-labelledby="subscribtion" name="emailInput" id="emailInput">
+              <button aria-labelledby="subscribtion" id="subscribeButton" name="subscribeButton" type="submit" class="button is-info">Notify Me</button>
+            </fieldset>
+          </form>
         </div>
       </div>
-    </div>
-  </div>
+    </section>
+  </header>
+	<main id="maincontent">
+		<div id="crumbs_frame" class="container" aria-label="breadcrumbs">
+			<ul class="bct">
+				<!-- .bct / Breadcrumbs -->
+				<li><a href="/">W3C</a><span class="cr">»</span></li>
```

Comment:
It looks like you can improve parts like this e.g.

```suggestion
<li><a href="./">MainPage</a><span class="cr">»</span></li>
<li><a href="index.html#nav">Pictures</a><span class="cr">»</span></li>
<li><a href="index.html#Services" class="current" aria-current="page">Services</a></li>
```

Reply:
This looks less abstract than mine. I will add it. Thank you.